### PR TITLE
feat(mock-render): option to detectChanges or not

### PR DESCRIPTION
**Pull request by ike18t**
#67 Accept an arg to allow control of detectChanges.

### CODE REVIEW

File: lib/mock-render/mock-render.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { directiveResolver } from '../common/reflect';
 
 function MockRender<MComponent, TComponent extends {[key: string]: any}>(
   template: string | Type<MComponent>,
-  params?: TComponent
+  params?: TComponent,
+  detectChanges = true
```
Comment (satanTime) [inline lib/mock-render/mock-render.ts]: maybe as an object in case if we need to add more flags later?

Comment (ike18t) [reply lib/mock-render/mock-render.ts]: I had that though too...but couldn't think of other options. My thought was that it was easy enough to write an overload/union if we were to pivot to an object.

Comment (satanTime) [reply lib/mock-render/mock-render.ts]: agree

Comment (ike18t) [reply lib/mock-render/mock-render.ts]: Welcome back BTW :-D

**Pull request merged**